boilingcore/templates.go

```diff
@@ -257,10 +263,10 @@ var templateStringMappers = map[string]func(string) string{
 
 var goVarnameReplacer = strings.NewReplacer("[", "_", "]", "_", ".", "_")
 
-// templateFunctions is a map of all the functions that get passed into the
+// boilerTemplateFunctions is a map of all the functions that get passed into the
 // templates. If you wish to pass a new function into your own template,
 // add a function pointer here.
```

Comment:
A little nitpicky, but I think it makes sense to update this comment too since it is no longer **all the functions**